pipelines/pipeline_util/pipeline_configuration.py

```diff
@@ -0,0 +1,11 @@
+class PipelineConfiguration:
+    """
+    Maintains settings which are treated as constants by the rest of
+    the pipeline.  Most of the settings refer to:
+        - which date format should be used for file names and the time
+          stamp format used in logging.
+        - the names of directories that appear when the pipelines are run
```

Comment:
Is this a stale comment, or are you intending to add directory names and CSV parameters later?

Reply:
This is a stale comment. I'll edit it.